backend/fms_core/template_importer/importers/sample_qc.py

```diff
@@ -0,0 +1,92 @@
+from fms_core.models import Process, Protocol, PropertyType
+
+from ._generic import GenericImporter
+from fms_core.template_importer.row_handlers.sample_qc import SampleQCRowHandler
+
+from .._utils import float_to_decimal_and_none, input_to_date_and_none
+
+# {{TEMPLATE PROPERTY NAME : DB PROPERTY NAME}
+TEMPLATE_PROPERTY_MAPPING = {
+    "Measured Volume (uL)": "Measured Volume",
+    "Concentration (ng/uL)": "Concentration",
+    "RIN (for RNA only)": "RIN",
+    "Electrophoresis Instrument": "Electrophoresis Instrument",
+    "Quantitation Instrument": "Quantitation Instrument",
+    "Comment": "Comment"
```

Comment:
Looking at the Process_Measurement detail page made me realize this property is not useful. Its a duplicate of the one stored directly on the process_measurement. Remove it from the properties. We'll store Comments for the QC directly on the process_measurement.